python/empty/main.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 # Quix injects credentials automatically to the client. Alternatively, you can always pass an SDK token manually as a parameter.

```

Comment:
Minor here (and all similar), could you replace "as a parameter" to "as an argument"?
Mostly they're interchangable and people will get it, but arg is what you pass in. It is called parameter in the function declaration.